crates/tessellation/src/fill.rs

```diff
@@ -2183,8 +2191,8 @@ impl<'l> FillVertex<'l> {
                     let a = store.get(id);
                     assert!(a.len() == num_attributes);
                     assert!(self.attrib_buffer.len() == num_attributes);
-                    for i in 0..num_attributes {
-                        self.attrib_buffer[i] += a[i];
+                    for (i, &att) in a.iter().enumerate().take(num_attributes) {
```

Comment:
This one is less readable. I'd prefer to `allow(clippy::needless_range_loop)` at the crate level since clippy isn't able to tell when the index is needed anyway.

Reply:
I just realized that `.take(num_attributes)` was redundant. Do you still stand by this?

Reply:
It's still meh, but alright nothing is set in stone anyway, I can always change it later if it rubs me the wrong way.